src/app/modules/verification/verification.controller.ts

```diff
@@ -87,28 +87,31 @@ export const handleGetTransactionMetadata: RequestHandler<
  * @param req
  * @param res
  */
-export const resetFieldInTransaction: RequestHandler<
+export const handleResetField: RequestHandler<
   { transactionId: string },
-  string,
+  { message: string },
```

Comment:
nit: I added a DTO type `ErrorDto` somewhere in the codebase to make it obvious what we are returning if you want to use it

Reply:
looks good, will make the change in #1455 to avoid merge conflicts